### DISCUSSION TITLE
Description comment for Revisions

### DISCUSSION THREAD

**Discussion by rizaardiyanto1412** [publishpress/PublishPress-Revisions#966] [Feature Requests]
> I was hoping that they were comments appended to the revision, so the person revising the post can leave some comments with an overview of what they did and why.
> we'd want submitters to be able to provide the comment before they submitted the revision. For example, a textarea field between the publish date and the Submit button that allowed the submitter to add a short description of what the revision was for.

**Comment by adriantoll** [comment]
This was my request. It would be great if the submitter could provide handoff notes to the reviewer so that the reviewer knows what the context of the revision is.
